### PR TITLE
test: add ut to test the input when creating evict-leader scheduler

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: Ref #4399

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: server/api/scheduler.go [hunk 1]
```diff
@@ -113,11 +113,16 @@ func (h *schedulerHandler) CreateScheduler(w http.ResponseWriter, r *http.Reques
 			return
 		}
 	case types.GrantLeaderScheduler, types.EvictLeaderScheduler:
-		storeID, ok := input["store_id"].(float64)
+		_, ok := input["store_id"]
```
Comment (rleungx) [inline server/api/scheduler.go:116]: Why?

Comment (okJiang) [reply server/api/scheduler.go:116]: `_, ok := input["store_id"]` represents the input("store_id") doesn't exist.

`storeID, ok := input["store_id"].(float64)` represents the input("store_id") can not be convert to float.

Comment (rleungx) [reply server/api/scheduler.go:116]: Is it necessary?

Comment (okJiang) [reply server/api/scheduler.go:116]: I think we can put it here. It won't be bad at all.

**Review by rleungx — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**